tests/Database/SelectFieldsTest.php

```diff
@@ -95,6 +97,40 @@ public function testWithSelectFieldsAndModel(): void
         $this->assertEquals($expectedResult, $response->json());
     }
 
+    public function testWithSelectFieldsAndModelAndAlias(): void
+    {
+        $post = factory(Post::class)->create([
+            'title' => 'Description of the post',
+        ]);
+
+        $graphql = <<<GRAQPHQL
+{
+  postWithSelectFieldsAndModelAndAlias(id: $post->id) {
+    id
+    description
+  }
+}
+GRAQPHQL;
+
+        $this->sqlCounterReset();
+
+        $response = $this->call('GET', '/graphql', [
+            'query' => $graphql,
+        ]);
+
+        $this->assertSqlQueries(<<<'SQL'
+select "posts"."id", "posts"."title" from "posts" where "posts"."id" = ? limit 1;
```

Comment:
Whats the issue with indenting this?

Reply:
Can you elaborate?

Reply:
its kinda painful to work this way! My editor went mad!
[Image](https://imgur.com/PsULGAc)

![img](https://imgur.com/download/PsULGAc)

Reply:
Hmm, works correctly for me => using PhpStorm 2019 here; what IDE are you using?

![collapse_expand](https://user-images.githubusercontent.com/87493/58687605-68474880-8382-11e9-814e-1415875db906.gif)

The issue I'm seeing:
- it doesn't matter for the GraphQL queries (whitespace is insignificant)
- but for the SQL assertion/comparison, they are significant 

Another reason I like to keep it this way is, especially for the SQL assertions, they _can_ get very long. Inevitably horizontal scrolling for them will be necessary often, but the less that is required, the better.